src/main/java/com/fvv/bookstore/util/StringUtil.java

```diff
@@ -0,0 +1,45 @@
+package com.fvv.bookstore.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to convert Strings
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 2018</p>
+ * @version 1.0 	
+ *
+ */
+public final class StringUtil {
+	
+	/**
+	 * Convert a List into a String.
+	 * 
+	 * @param listProducts of List type.
+	 * @return a String of products' preference 
+	 */
+	public static String convertListToString(final List<String> listProducts) {
+		String preferences = "";
+		for(String pref : listProducts) {
+			preferences += pref + Constants.PIPE;
```

Comment:
Here it applies the theory about how a String is created we discussed earlier, for this reason, avoid using String concatenation here. Instead, use a StringBuilder and its method append like you already did before

Reply:
Ok